.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     rev: 21.9b0
     hooks:
       - id: black
-        language_version: python3.9
+        language_version: python3.8
```

Comment:
@reiktar Shouldn't this be python3.9?

Reply:
@exemaitch: this is the version of python used to run black. Black still validates 3.9 codes as expected. 

This is a workaround for an issue where black in somecases does not work with the python 3.9 runtime.

Reply:
@exemaitch: Testing 3.9 again .. was 18 days since this was tested last

Reply:
@exemaitch 
the following error appears when setting the build number to 3.9 

https://github.com/xmlabs-io/exemaitch/runs/4219209708?check_suite_focus=true

Reply:
@exemaitch  This issue seeems to recmmend using python3 instead of python3.X

https://github.com/pre-commit/pre-commit/issues/1375